static/app/components/environmentPageFilter.tsx

```diff
@@ -13,15 +13,24 @@ import space from 'sentry/styles/space';
 import useOrganization from 'sentry/utils/useOrganization';
 import useProjects from 'sentry/utils/useProjects';
 
-type Props = {
+type Props = Omit<
+  React.ComponentProps<typeof MultipleEnvironmentSelector>,
```

Comment:
is the main prop we need to get from MultipleEnvironmentSelector just the `alignDropdown` prop? Seems a little verbose to take all the component's props and omit a lot of them.

Reply:
hmm I originally intended to pass all props down to <MultipleEnvironmentSelector /> (hence the `…props` spread below this), but you're right it doesn't seem necessary at this point - just updated the component to only pass `alignDropdown` down.